src/components/AgreementSections/LinesList.js

```diff
@@ -57,7 +57,18 @@ export default class LinesList extends React.Component {
     type: line => <EResourceType resource={getResourceFromEntitlement(line)} />,
     count: line => <EResourceCount resource={getResourceFromEntitlement(line)} />,
     coverage: line => <CoverageStatements statements={line.coverage} />,
-    isCustomCoverage: line => (line.customCoverage ? <CustomCoverageIcon /> : ''),
+    isCustomCoverage: line => {
+      if (!line.customCoverage) return '';
+      return (
+        <Tooltip
+          id="custom-coverage-tooltip"
```

Comment:
Just realised that this `id` won't be unique across the page while it should. So maybe something like this:
```suggestion
          id={`agreement-line-cc-tooltip-${line.rowIndex}`}
```

(`rowIndex` is provided by `MultiColumnList`)